### ISSUE TITLE
Problem: We need a upgrade path/process from Airtime 2.5.x 

### ISSUE THREAD

**Issue by Robbt** [LibreTime/libretime#71]
This might be tricky but it would be good to document the process for upgrading from Airtime 2.5.x since most of the database update scripts are included this might not be too difficult. This shouldn't be a priority until we get out of alpha though because we don't want people switching their production machines until we've done more testing.

**Comment by hairmare** [comment]
Yes, we should get this up and running when we get to beta.

I don't believe that anything broke with the process, we should figure out from what versions people will be upgrading. I'm pretty sure that upgrading from 2.5 works but I also assume that more folks are on pre-2.5 since 2.5 was never a very nice release to begin with.

**Comment by Robbt** [comment]
I have succesfully upgraded a 2.5.x system at our remote transmitter site and it went rather smoothly.

**Comment by hairmare** [comment]
I recently ran the database migrations against a dump of our production database and didn't have any issues either.

**Comment by hairmare** [comment]
See #188

**Comment by hairmare** [comment]
There's still the issue about media monitor not being removed so I'm bumping this to the next milestone.

**Comment by Robbt** [comment]
Considering the idea that a beta version of LibreTime should be a plug-in replacement for Airtime 2.5.1 debian based instances of Airtime this should be solved and tested before we consider ourselves a beta.

**Comment by b01xy** [comment]
[an issue](https://github.com/LibreTime/libretime/issues/771) about upgrading from airtime 2.5.1 to Libretime 3.0.0. alpha

**Comment by Robbt** [comment]
So I was able to do a straight ./install -fiap after cloning the git master and everything appeared to be working on an airtime to libretime upgrade. I installed with the debian files provided by sourcefabric onto airtime 14.04.

Anyone coming from Airtime at this point is probably going to need to deal with a distro upgrade to at least 16.04 or the latest debian as well. 

Once we have a deb file for the latest release I'll test upgrading by installing the deb package @paddatrapper builds and see if it has any issues,
